packages/react-components/react-persona/Spec.md

```diff
@@ -1,63 +1,223 @@
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+![Avatar](./etc/images/avatar.png)
+
+Persona with Avatar:
+
+```jsx
+<Persona name="Kevin Sturgis" primaryText="Kevin Sturgis" secondaryText="Software Engineer" />
+```
+
+![PresenceBadge](./etc/images/badge.png)
+
+Persona with PresenceBadge:
+
+```jsx
+<Persona presenceOnly presence={{ status: 'offline', outOfOffice: true }} primaryText="Kevin Sturgis" />
+```
+
+![Avatar and PresenceBadge](./etc/images/avatar_badge.png)
+
+Persona with Avatar + PresenceBadge:
+
+```jsx
+<Persona
+  name="Kevin Sturgis"
+  presence={{ status: 'offline', outOfOffice: true }}
+  primaryText="Kevin Sturgis"
+  secondaryText="Software Engineer"
+  tertiaryText="Offline"
+/>
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+> I will refer to the avatar and presence as content.
+
+There are three alignment variants:
+
+- after: Content on the left and text on the right.
+- before: Content on top and text on the bottom.
+- below: Content on the right and text on the left.
+
+There are 3 content variants:
+
+- Avatar
+- PresenceBadge
+- Avatar + PresenceBadge
+
+There are 2 sizing variants:
+
+- scaled: The content will be resized based on the number of text lines.
+- fixed: The content will remain the same size and the text lines will be styled based on the content's size.
 
 ## API
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+**Slots**
+
+- `root`: The root slot for Persona.
+- `avatar`: The Avatar to display and Persona's primary slot.
+- `presence`: The PresenceBadge to display.
+- `primaryText`: Primary text.
+- `secondaryText`: Secondary text.
+- `tertiaryText`: Tertiary text.
+- `quaternaryText`: Quaternary text.
+
+**Types**
+
+```ts
+export type PersonaSlots = {
+  root: NonNullable<Slot<'div'>>;
+
+  /**
+   * Avatar to display.
+   */
+  avatar: NonNullable<Slot<typeof Avatar>>;
+
+  /**
+   * PresenceBadge to display.
+   */
+  presence: Slot<typeof PresenceBadge>;
+
+  /**
+   * Primary text to display.
+   */
+  primaryText: Slot<'span'>;
+
+  /**
+   * Secondary text to display.
+   */
+  secondaryText: Slot<'span'>;
+
+  /**
+   * Tertiary text to display.
+   */
+  tertiaryText: Slot<'span'>;
+
+  /**
+   * Quaternary text to display.
+   */
+  quaternaryText: Slot<'span'>;
+};
+
+/**
+ * Persona Props
+ */
+export type PersonaProps = Omit<ComponentProps<Partial<PersonaSlots>, 'avatar'>, 'badge'> & {
+  /**
+   * Whether to display only the presence.
+   *
+   * @default false
+   */
+  presenceOnly?: boolean;
+
+  /**
+   * Sizing type to use.
+   *
+   * `fixed`: Text lines adjust to content's size.
+   * `scaled`: Content adjusts its size based on the number of text lines used.
+   *
+   * @default fixed
+   */
+  sizing?: 'fixed' | 'scaled';
```

Comment:
I don't think we need this `sizing` prop, since it's redundant with `size`:
* When `sizing="fixed"`, then it requires specifying the `size` prop (or rely on the default 32).
* When `sizing="scaled"`, then you can't use the `size` prop.

Therefore, we can determine which one to use just by whether or not the `size` prop was supplied.

Or, another way to describe it: the default value of the `size` prop is based on the number of lines of text (the "scaled" variant), but can be overridden by specifying the size (the "fixed" variant).